helmfile/charts/grafana-ops/files/welcome.md

```diff
@@ -1,5 +1,12 @@
 # Welcome to Compliant Kubernetes!
 
+## What's new
+
+Here you can find the most relevant features and changes for the last couple of releases of Compliant Kubernetes
+
+- As a User admin, you can now create namespaces yourself using [HNC](https://kubernetes.io/blog/2020/08/14/introducing-hierarchical-namespaces/), we have written a guide [here](https://elastisys.io/compliantkubernetes/user-guide/faq/#how-do-i-add-a-new-namespace) on how you do this in Compliant Kubernetes.
```

Comment:
```suggestion
Here you can find the most relevant features and changes for the last couple of releases of Compliant Kubernetes

#### Release v0.25
- As a User admin, you can now create namespaces yourself using [HNC](https://kubernetes.io/blog/2020/08/14/introducing-hierarchical-namespaces/), we have written a guide [here](https://elastisys.io/compliantkubernetes/user-guide/faq/#how-do-i-add-a-new-namespace) on how you do this in Compliant Kubernetes.
```

can we also add the release version? it would be easier for us to know what to remove

Reply:
Hmm maybe we can put a comment at the end of the point to tell which version it is from, I don't want this to look like a release notes page

Reply:
Made some changes to clarify this, PTAL